v2.1/performance-benchmarking.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Performance Benchmarking
```

Comment:
Is it important to call out TPC-C in the title? If so, what about `Performance Benchmarking with TPC-C`?